lib/vmpooler/providers/vsphere.rb

```diff
@@ -165,7 +165,7 @@ def get_vm(_pool_name, vm_name)
           vm_hash = nil
           @connection_pool.with_metrics do |pool_object|
             connection = ensured_vsphere_connection(pool_object)
-            vm_object = find_vm(vm_name, connection)
+            vm_object = find_vm(_pool_name, vm_name, connection)
```

Comment:
The ruby convention of underscore prefix is denote variables which aren't used.  As this variable is not being used, the underscore should be removed.

Reply:
Thanks, I'll fix this and push a change.